app/Http/Controllers/UsersController.php

```diff
@@ -344,6 +344,19 @@ public function togglePrivacy (Request $request)
         $user->save();
     }
 
+    /**
+     * Toggle a Users tagging settings
+     */
+    public function toggleTagging (Request $request)
+    {
+        $user = Auth::user();
+
+        /* Whether to save the tags from a previous image */
+        $user->previous_tags = $request->previous_tags;
+
+        $user->save();
```

Comment:
We should also return a response and handle it 

```php
return [
    'success' => true
];
```

```javascript

.then(response => {
   console.log('my_action', response); 
   
    if (response.data.success)
    {
          // Handle success eg show successful event notification
          
```

Reply:
good catch